operator/gitops/argocd/grafana/dashboards/pipeline-service-dashboard.json

```diff
@@ -167,10 +160,6 @@
       "type": "stat"
     },
     {
-      "datasource": {
```

Comment:
I believe this was part of the problem with the Alert panels not rendoring immediately